template.yml

```diff
@@ -38,3 +38,15 @@ Resources:
             Properties:
               Path: /v3/technical-records
               Method: post
+    PatchLambdaFunction:
+      Type: 'AWS::Serverless::Function'
+      Properties:
+        CodeUri: src/handler/
+        Handler: archive.handler
+        Runtime: nodejs18.x
+        Events:
+          PostLambdaApi:
+            Type: Api
+            Properties:
+              Path: /v3/technical-records/archive/{systemNumber}/{createdTimestamp}
+              Method: patch
```

Comment:
This is a patch because the technical record is no longer required as part of the body. In V1 it was and it wasn't great for security because it opened up the possibility of being able to misuse the endpoint as a general purpose tech record replacement. The endpoint now retrieves this from the database and only changes the required fields to archive the record, this makes it a PATCH operation rather than a PUT operation which it really should have been in the first place.